### DISCUSSION TITLE
Delta Feed

### DISCUSSION THREAD

**Discussion by eeadie** [OData/AspNetCoreOData#1373] [Q&A]
Hi,

Is there an example of implementing a Delta Feed with AspNetCoreOData? Similar to the following:

[Delta Feed Support](https://learn.microsoft.com/en-us/odata/webapi/deltafeed_support)

Thanks,

Euan

**Comment by gathogojr** [comment]
@eeadie It's pretty much the same as described in the doc you referenced. Go ahead and give it a try
